packages/cli/src/databases/entities/folder.ts

```diff
@@ -0,0 +1,48 @@
+import {
+	Column,
+	Entity,
+	JoinColumn,
+	JoinTable,
+	ManyToMany,
+	ManyToOne,
+	OneToMany,
+} from '@n8n/typeorm';
+import { IsString, Length } from 'class-validator';
+
+import { WithTimestampsAndStringId } from './abstract-entity';
+import { Project } from './project';
+import { TagEntity } from './tag-entity';
+import { type WorkflowEntity } from './workflow-entity';
+
+@Entity()
+export class Folder extends WithTimestampsAndStringId {
+	@Column({ length: 128 })
+	@IsString({ message: 'Folder name must be of type string.' })
+	@Length(1, 128, { message: 'Folder name must be $constraint1 to $constraint2 characters long.' })
```

Comment:
We don't necessary need these, as we should use `zod` schemas to define the constraints

Reply:
I agree here. we need to start removing all such validation checks in entities, and make sure that all user input is validated in DTOs.